src/Network/Ipfs/Api/Api.hs

```diff
@@ -0,0 +1,802 @@
+{-# LANGUAGE DataKinds             #-}
+{-# LANGUAGE TypeOperators         #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+
+-- |
+-- Module      :  Network.Ipfs.Api.Api
+-- Copyright   :  Alexander Krupenkin 2016-2018
+-- License     :  BSD3
+--
+-- Maintainer  :  mail@akru.me
+-- Stability   :  experimental
+-- Portability :  portable
+--
+-- Ipfs API provider.
+--
+
+module Network.Ipfs.Api.Api where
+
+import           Control.Arrow                 (left)
+import           Control.Monad
+import           Data.Aeson
+import           Data.Int
+import           Data.ByteString.Lazy          (toStrict)
+import qualified Data.ByteString.Lazy.Char8()
+import qualified Data.HashMap.Strict           as H
+import           Data.Proxy           
+import qualified Data.Text                     as TextS
```

Comment:
So many `TextS` prefixes, why you don't just import `Text` type?

Reply:
Agreed, I'll change it.